README.md

```diff
@@ -55,9 +55,8 @@ Create an index.html file (see [docs](/docs/root-application.md#indexhtml-file)
 Create the root application (see [docs](/docs/root-application.md) for more detail).
 ```js
 // src/main.js
-import { declareChildApplication, setLoader, start } from "single-spa";
-setLoader(SystemJS);
-declareChildApplication('/apps/app1/app1.js', () => window.location.hash === '');
+import { declareChildApplication, start } from "single-spa";
+declareChildApplication('/apps/app1/app1.js', () => System.import('/apps/app1/app1.js'), () => window.location.hash === '');
```

Comment:
As we discussed offline, it's for `getAppStatus` and `getMountedApps` apis. But I'll change this example so it's clear what the difference is between the name and doLoad.